sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/OptimizeJsonExprs.scala

```diff
@@ -34,8 +35,10 @@ import org.apache.spark.sql.types.{ArrayType, StructType}
  *      contains all accessed fields in original CreateNamedStruct.
  */
 object OptimizeJsonExprs extends Rule[LogicalPlan] {
+  private def enabledJsonOptimization = SQLConf.get.jsonExpressionOptimization
```

Comment:
We need this variable? Could we inline this in L41?

Reply:
Sure.